runtime/vm/romclasses.c

```diff
@@ -247,7 +247,7 @@ internalCreateBaseTypePrimitiveAndArrayClasses(J9VMThread *currentThread)
 				NULL, /* protectionDomain */
 				NULL, /* methodRemapArray */
 				J9_CP_INDEX_NONE, /* entryIndex */
-				LOAD_LOCATION_UNKNOWN, /* locationType */
+				LOAD_LOCATION_MODULE, /* locationType */
```

Comment:
This doesn't seem right for Java 8.

Reply:
would `LOAD_LOCATION_CLASSPATH` be correct for java8?

Reply:
I don't think so as that would imply that the classfile was loaded from the class path.  Thinking about it further, I'm not certain `LOAD_LOCATION_MODULE` is correct for 9 as primitives are generated by the VM.

Reply:
Good point.

Reply:
The other option is to keep it as `LOAD_LOCATION_UNKNOWN` and add a condition in createramclass to add primitives to java.base instead of the unnamedModule